### PR TITLE
IRO-1701 - Added Block information page

### PR DESCRIPTION
**Pull request by ndemidovich-sc**

### CODE REVIEW

File: components/BlocksTable/BlocksTable.tsx
Comment (brekk) [inline components/BlocksTable/BlocksTable.tsx:50]: Not a blocker, but we should use `date-fns` for this eventually.

File: components/Breadcrumbs/BreadcrumbLink.tsx
Comment (brekk) [inline components/Breadcrumbs/BreadcrumbLink.tsx:87]: Might make sense to name this partial type, since it is reused a bunch?

```ts
type CurrentBreadcrumbLink = Pick<BaseBreadcrumbLinkProps, 'isCurrent'>
```

File: components/Table/DataRowLarge.tsx
Comment (brekk) [inline components/Table/DataRowLarge.tsx:7]: What's the identity function doing here?

File: components/Table/DataRowSmall.tsx
Comment (brekk) [inline components/Table/DataRowSmall.tsx:21]: Same

File: components/Table/Table.tsx
Comment (brekk) [inline components/Table/Table.tsx:25]: We should avoid using `any`. I'll add this to a subsequent PR, so not a blocker here.

File: components/Table/DataRowSmall.tsx
Comment (brekk) [inline components/Table/DataRowSmall.tsx:19]: Seeing a: `validateDOMNesting(...): <div> cannot appear as a child of <tbody>` warning on local and I think this element might be the culprit. I believe you can get around it by adding `as="tr"` but I'm not certain (see [this](https://chakra-ui.com/docs/components/recipes/as-prop) for more)

File: components/Table/RowItem.tsx
Comment (brekk) [inline components/Table/RowItem.tsx:11]: Probably `as="tr"` here as well

File: pages/blocks/[id].tsx
Comment (brekk) [inline pages/blocks/[id].tsx:77]: Where does this function get used?

File: services/TransactionService.ts
Comment (brekk) [inline services/TransactionService.ts:18]: Why's this changing?

**Review by brekk — CHANGES_REQUESTED** [review]
I'd really like to get this out, but I found a few issues both in review and running locally (I think some of these can be easily addressed with the `as` prop) but I'd like to see them get fixed before this merges.

**Comment by brekk** [comment]
I created two PRs that are cut from this branch:

1. [Fix fonts](https://github.com/iron-fish/block-explorer-v2/pull/18) - Pointing at this branch
2. [Update and enforce eslint](https://github.com/iron-fish/block-explorer-v2/pull/19) - Pointing at the `update-fonts` branch

@ndemidovich-sc If it makes sense please feel free to merge 2 into 1 and 1 into this PR. Or we can do it after this PR merges, I'm open.

File: components/Breadcrumbs/BreadcrumbLink.tsx
Comment (azhurauski-sc) [reply components/Breadcrumbs/BreadcrumbLink.tsx]: updated

File: components/Table/DataRowLarge.tsx
Comment (brekk) [inline components/Table/DataRowLarge.tsx:7]:
```suggestion
    {items}
```

File: components/Table/DataRowSmall.tsx
Comment (brekk) [inline components/Table/DataRowSmall.tsx:21]:
```suggestion
      {items}
```

**Review by brekk — APPROVED** [review]
Minor fixables suggested, otherwise I think this can go out.

**Review by skylarrichard123 — APPROVED** [review]
Approved provided the navigation issue mentioned in slack can be separated from this

EDITED:

Mobile padding issue's here https://linear.app/ironfish/issue/IRO-1999/mobile-padding

**Review by skylarrichard123 — CHANGES_REQUESTED** [review]
https://linear.app/ironfish/issue/IRO-1999/mobile-padding

**Review by skylarrichard123 — APPROVED** [review]

**Pull request merged**
